package.json

```diff
@@ -50,6 +52,7 @@
     "sass-loader": "3.1.2",
     "striptags": "2.1.1",
     "style-loader": "0.13.0",
+    "url-loader": "0.5.7",
```

Comment:
check `loaders` in the webpack config, it's for including static assets like images and stuff.

here:

![loaders](https://cloud.githubusercontent.com/assets/6207644/14825999/0a3bc3ea-0baa-11e6-9abb-303e05e1d50e.png)